pkg/apis/applicationmonitoring/v1alpha1/applicationmonitoring_types.go

```diff
@@ -21,6 +21,7 @@ type ApplicationMonitoringSpec struct {
 	PrometheusInstanceNamespaces     string `json:"prometheusInstanceNamespaces,omitempty"`
 	AlertmanagerInstanceNamespaces   string `json:"alertmanagerInstanceNamespaces,omitempty"`
 	SelfSignedCerts                  bool   `json:"selfSignedCerts,omitempty"`
+	PodAntiAffinity                  bool   `json:"PodAntiAffinity,omitempty"`
```

Comment:
```suggestion
	PodAffinity                  *corev1.Affinity   `json:"PodAffinity,omitempty"`
```

I wonder if it would be more flexible to follow the pattern used by 3scale [1] and include the entire affinity rules in the CR. This would allow us to have the logic to decide which rules to use in the integreatly operator, and if there's any changes they would be made there

[1] https://github.com/3scale/3scale-operator/blob/master/pkg/apis/apps/v1alpha1/apimanager_types.go#L176

Reply:
That's definitely a better approach.